quizroyale/server/login_server.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+
+import jwt
+from flask import Flask, request, render_template, jsonify
+
+from quizroyale.db.db import DummyDbService
+from quizroyale.db.errors import NoSuchUserException
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello_world():
+    return render_template('login.html')
```

Comment:
This can stay for now, we won't be using templates in the future. We will handle dummy requests in a different way. I'll implement that later

Reply:
Yes, I know. This is temporary solution